components/d2l-organization-date/lang/fr-on.js

```diff
@@ -0,0 +1,5 @@
+export default {
+	"ended": "S\'est terminé le {date} à {time}",
```

Comment:
```suggestion
	"ended": "S'est terminé le {date} à {time}",
```
Should be able to remove the escaping as well

Reply:
I've removed the escaping with your Git suggestion, thanks!  My locally generated translations seemed to work fine with the `\` in place, but I'll happily remove it.
```
    <langTerm sortOrder="1" name="ended">
      <defaultValue><![CDATA[S'est terminé le {date} à {time}]]></defaultValue>
      <description><![CDATA[]]></description>
    </langTerm>
```